test/test_profiler.py

```diff
@@ -1127,6 +1128,20 @@ def test_utils_compute_self_time(self):
                     for child in event_key.event.children
                 ]))
 
+    @unittest.skipIf(not torch.cuda.is_available(), "CUDA is required")
+    def test_utils_compute_queue_depth(self):
+        x = torch.ones((4096, 4096), device="cuda")
+        with profile() as prof:
+            for _ in range(5):
```

Comment:
You should add a comment about why the ops are structured this way, as it's not obvious why we would synchronize and sleep.